pipeline_dp/pipeline_operations.py

```diff
@@ -349,3 +346,218 @@ def count_per_element(self, col, stage_name: typing.Optional[str] = None):
 
     def reduce_accumulators_per_key(self, col, stage_name: str = None):
         return self.map_values(self.group_by_key(col), accumulator.merge)
+
+
+# workaround for passing lambda functions to multiprocessing
+# according to https://medium.com/@yasufumy/python-multiprocessing-c6d54107dd55
+_pool_current_func = None
+
+
+def _pool_worker_init(func):
+    global _pool_current_func
+    _pool_current_func = func
+
+
+def _pool_worker(row):
+    return _pool_current_func(row)
+
+
+class _LazyMultiProcIterator:
+    def __init__(
+        self,
+        job: typing.Callable,
+        job_inputs: typing.Iterable,
+        chunksize: int,
+        n_jobs: typing.Optional[int],
+        **pool_kwargs
+    ):
+        """Utilizes the `multiprocessing.Pool.map` for distributed execution of 
+        a function `job` on an iterable `job_inputs`.
+
+        Parameters
```

Comment:
Nit: according to [Google style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) (and for consistency with other functions), the format of function comments:
Args:
   ...